spec/std/crystal/compiler_rt/divmod128_spec.cr

```diff
@@ -0,0 +1,93 @@
+require "spec"
+
+# TODO: Replace helper methods with literals once possible
+
+private def make_ti(a : Int128, b : Int128)
+  (a << 64) + b
+end
+
+private def make_tu(a : UInt128, b : UInt128)
+  (a << 64) + b
+end
+
+# Ported from:
+# - https://github.com/llvm/llvm-project/blob/main/compiler-rt/test/builtins/Unit/umodti3_test.c
```

Comment:
Please, use the exact commit sha instead of `main`.

ditto elsewhere

Reply:
The files that were ported haven't changed in a long time.
Do you think it would also be okay to use release/13.0 instead of main?
I think that's more expressive